.github/workflows/deployment-test-template.yml

```
@@ -17,9 +17,27 @@ jobs:
         shell: bash
         env:
           DATABASE_PASSWORD_FILE: "./db_password_example.conf"
+          BACK_ENV_FILE: "./junqo_back/exemple.env"
         run: docker compose -f ${{ inputs.compose_file }} up -d --build
 
+      - name: Check Container Status
+        if: always()
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2162:info:4:9: read without -r will mangle backslashes

(shellcheck)